lib/v2/leetcode-com/contest.js

```diff
@@ -0,0 +1,35 @@
+const got = require('@/utils/got');
+const moment = require('moment');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://leetcode.com';
+    const currentUrl = `${rootUrl}/graphql`;
+    const response = await got({
+        method: 'post',
+        url: currentUrl,
+        json: {
+            query: '{\n  brightTitle\n  currentTimestamp\n  allContests {\n    containsPremium\n    title\n    cardImg\n    titleSlug\n    description\n    startTime\n    duration\n    originStartTime\n    isVirtual\n    company {\n      watermark\n      __typename\n    }\n    __typename\n  }\n}\n',
+        },
+    });
+    const allContests = response.data.data.allContests;
+
+    const contestBaseUrl = 'https://leetcode.com/contest/';
+    const items = allContests.slice(0, 10).map((contest) => {
+        const beijingTime = moment
+            .utc(contest.startTime * 1000)
+            .utcOffset('+0800')
+            .format('YYYY-MM-DD HH:mm:ss');
+        const duration = contest.duration / 3600.0;
```

Comment:
I think you can also use day.js to achieve which is already available in current package.json